### ISSUE TITLE
18 related safe address

### ISSUE THREAD

**Issue by STephcuYY** [safe-global/safe-user-allocation-reports#159]
# Related Safe Addresses
0x29c324e806047075b179dc0d21ae27612efc2555
0x162cebe7efe040c16001656e9a10b5ee7fdd06a7
0x57d304a89543fffa13bc16e8edb2943fef86ce99
0x7331c2363bfafa459a09c8c8a694911208cb680f
0xa2f5acba956a226e91c87b0f4c2215534332fa5b
0x985abcb183a7ce2d85bc7528f1d3d94aecfd4a79
0x4d729d5d332afe2e0ba5e88d3c47ab772f83169c
0xe51bbd1ad216bba7dc7e98209fd37238dfcb5a9c
0x906a96fb54fdf91c04097253410d35ec2dec283f
0x9952401dc02b7332e709c2c7a6b8d5a5819272d7

# Reasoning
all created by 0x405575eca1422b464c539e4db9f639b85d0b8e9f
![2](https://user-images.githubusercontent.com/113139158/189215480-910a155b-a64e-4c90-b3ac-c2444ea6c619.png)
And he used to operate 18 safe addresses at the same time in a day, which is obviously the same airdrop farmer doing meaningless farming on a regular basis.
In addition it can't be a repeater, because it only has 145 tx in existence.
To sum up, this is a farmer who operates 18 safes at the same time

# Methodology
with target as (
select *
from url('https://raw.githubusercontent.com/safe-global/safe-user-allocation-reports/main/safe_user_allocations_reworked.csv', CSVWithNames)
), transactions as (
select t1.*
from ethereum.transactions t1
join target t2
on t1.to_address = t2.safe_address
)

select distinct to_address
from transactions
where from_address = lower('0x405575eca1422b464c539e4db9f639b85d0b8e9f')

# Safe Address
0x022915d31115406A37E55092D4298A711aC056Ac
